support/developer/biztalk/management-operations/maintain-troubleshoot-database.md

```diff
@@ -189,23 +191,23 @@ Additionally, you can use the BHM tool output to determine which tables are the
 
 |Table|Description|
 |---|---|
-| **HostName** Q_Suspended|This table contains a reference to messages in the Spool table that are associated with suspended instances for the particular host. This table is in the BizTalkMsgBoxDb database.|
-| **HostName** Q|This table contains a reference to messages in the Spool table that are associated with the particular host and are not suspended. This table is in the BizTalkMsgBoxDb database.|
-|Spool<br/><br/>Parts<br/><br/>Fragments|These tables store actual message data in the BizTalkMsgBoxDb database.|
-|Instances|This table stores all instances and their current status in the BizTalkMsgBoxDb database.|
-|TrackingData_0_ _x_|These four tables store the Business Activity Monitoring (BAM) tracked events in the BizTalkMsgBoxDb database for TDDS to move the events to the BAMPrimaryImport database.|
-|TrackingData_1_ _x_|These four tables store the tracked events in the BizTalkMsgBoxDb database for TDDS to move the events to the BizTalkDTADB database.|
-|Tracking_Fragments **x** <br/>Tracking_Parts **x** <br/>Tracking_Spool **x**|Two of each of these tables is in the BizTalkMsgBoxDb and BizTalkDTADb databases. One is online, and the other is offline.<br/><br/>In BizTalk Server 2004 SP2 and in later versions, the TrackedMessages_Copy_BizTalkMsgBoxDb SQL Server Agent job moves tracked message bodies directly to these tables in the BizTalkDTADb database.<br/><br/>In BizTalk Server 2004 Service Pack 1 (SP1) and in earlier versions of BizTalk Server 2004, the TrackedMessages_Copy_BizTalkMsgBoxDb SQL Server Agent job copies tracked message bodies into these tables in the BizTalkMsgBoxDb database. The TrackingSpool_Cleanup_BizTalkMsgBoxDb SQL Server Agent job purges the offline tables and makes the tables online while the job also takes the online tables offline.|
-|dta_ServiceInstances|This table stores tracked events for service instances in the BizTalkDTADb database. If this table is large, the BizTalkDTADb database is probably large.|
-|dta_DebugTrace|This table stores the Orchestration debugger events in the BizTalkDTADb database.|
-|dta_MessageInOutEvents|This table stores tracked event messages in the BizTalkDTADb database. These tracked event messages include message context information.|
-|dta_ServiceInstanceExceptions|This table stores error information for any suspended service instance in the BizTalkDTADb database.|
+|HostName `Q_Suspended`|This table contains a reference to messages in the Spool table that are associated with suspended instances for the particular host. This table is in the `BizTalkMsgBoxDb` database.|
```

Comment:
Not the format that we confirmed offline. If it's a placeholder, use braket for it.

Reply:
I am not sure if it is a placeholder.  I have added the code format just in case. @aartig13, could you please direct us to a person who can confirm this?